packages/@react-spectrum/color/docs/ColorSwatch.mdx

```diff
@@ -63,14 +63,6 @@ function Example() {
 }
 ```
 
-### No value
```

Comment:
So how do you get this state then? Pass in a transparent color? Should we have an example of that?